test/tests/physics/diffusion_cg.i

```diff
@@ -0,0 +1,89 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '1 2'
+    dy = '2 1'
+    ix = '2 3'
+    iy = '3 2'
+    subdomain_id = '0 1
+                    1 0'
+  []
+  [split_boundaries]
+    type = BreakBoundaryOnSubdomainGenerator
+    input = cmg
+  []
+  allow_renumbering = false
```

Comment:
why?

Reply:
NodalValue sampler cant be trusted with distributed mesh otherwise